onnx/defs/schema.h

```diff
@@ -506,7 +483,7 @@ class OpSchemaRegistry final {
       // Increase the highest version when you make BC-breaking changes to the
       // operator schema on specific domain. Update the lowest version when it's
       // determined to remove too old version history.
-      map_[ONNX_DOMAIN] = std::make_pair(1, 6);
+      map_[ONNX_DOMAIN] = std::make_pair(1, 7);
```

Comment:
It has been bumped from 5 to 6 for this purpose https://github.com/onnx/onnx/commit/5f69c37628002efa8c03d70d652db03c9d5ffca7